source/_integrations/enocean.markdown

```diff
@@ -26,7 +26,16 @@ related:
 
 The [EnOcean](https://en.wikipedia.org/wiki/EnOcean) standard is supported by many different vendors. There are switches and sensors of many different kinds, and typically they employ energy harvesting to get power such that no batteries are necessary.
 
-The EnOcean integration adds support for some of these devices. You will need a controller like the [USB300](https://www.enocean.com/product/usb-300/) in order for it to work.
+The EnOcean integration adds support for some of these devices. You will need a controller like the [USB300](https://www.enocean.com/product/usb-300/) in order for it to work. 
+
+## Unsupported controllers/gateways
+
+The following gateways are not supported by the integration:
+- All gateways that are based on the ESP2 protocol.  You can find an excellent guide to controllers and suitable integrations at this [gateway overview](https://github.com/grimmpp/home-assistant-eltako/blob/main/docs/gateways/readme.md).
```

Comment:
We shouldn't be sending users to library documentation. If this is relevant for the user reading this page, we should include it here.

../Frenck

Reply:
Let's remove that link in that case, as the main comment still remain :D